packages/browsers/README.md

```diff
@@ -2,6 +2,13 @@
 
 Manage and launch browsers/drivers from a CLI or programmatically.
 
+## System requirements
+
+- Node: Latest LTS or newer versions.
```

Comment:
This line sound wrong, we currently support v18 as it still has Security updates.
Better reference the engines part in the package.json